### PR TITLE
Organize with folders for better readability

### PR DESCRIPTION
**Pull request by BrunoKrugel**
wip

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+tags
```
Comment (lucario387) [inline .gitignore]: Is there a reason for this gitignore? I don't think it's relevant here

Comment (BrunoKrugel) [reply .gitignore]: ctags, it's just to make easier, otherwise I have to stash my ctags file every time before push

Comment (lucario387) [reply .gitignore]: You can commit specific files by doing `git commit lua/` though no?

**Review by siduck — APPROVED** [review]

**Review by lucario387 — APPROVED** [review]
Let it fly :airplane:

**Pull request merged**